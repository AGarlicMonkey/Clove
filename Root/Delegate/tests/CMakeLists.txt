#SingleCastDelegate
add_executable(SingleCastDelegateTest SingleCastDelegateTests.cpp)
target_link_libraries(SingleCastDelegateTest PRIVATE gtest_main Root.Delegate)
add_test(NAME SingleCastDelegateTest COMMAND SingleCastDelegateTest)

#MultiCastDelegate
add_executable(MultiCastDelegateTest MultiCastDelegateTests.cpp)
target_link_libraries(MultiCastDelegateTest PRIVATE gtest_main Root.Delegate)
add_test(NAME MultiCastDelegateTest COMMAND MultiCastDelegateTest)