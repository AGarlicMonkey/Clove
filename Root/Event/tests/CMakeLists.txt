add_executable(EventTest EventTests.cpp)
target_link_libraries(EventTest PRIVATE gtest_main Root.Event)
add_test(NAME EventTest COMMAND EventTest)