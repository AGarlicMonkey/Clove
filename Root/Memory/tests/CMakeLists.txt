#StackAllocator
add_executable(StackAllocatorTest StackAllocatorTests.cpp)
target_link_libraries(StackAllocatorTest PRIVATE gtest_main Root.Memory )
add_test(NAME StackAllocatorTest COMMAND StackAllocatorTest)