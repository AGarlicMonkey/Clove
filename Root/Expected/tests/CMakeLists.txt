add_executable(ExpectedTest ExpectedTests.cpp)
target_link_libraries(ExpectedTest PRIVATE gtest_main Root.Expected)
add_test(NAME ExpectedTest COMMAND ExpectedTest)